scripts/reset_tools.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+ssh -i your_staging-ffxbld_private_key_location stage-ffxbld@hg.mozilla.org edit tools 1 YES
```

Comment:
For future, we might want to grab that key via cmd args but for now it's super useful as is 👍